OMCompiler/toJulia/tokens.jl

```diff
@@ -1,15 +1,19 @@
+using MetaModelica : ModelicaReal
+
 abstract type Token end
 struct FileInfo
   line::Int
   column::Int
 end
-
 struct IntToken <: Token
-  value::Int
+  value::Signed
 end
 struct RealToken <: Token
   original::String
-  value::Real
+  value::ModelicaReal
+end
+struct BoolToken <: Token
```

Comment:
@sjoelund  When I added this I thought it was necessary, but everything is  still more or less WIP

Reply:
Probably not necessary as it's just tokens